schemas/com.fieldnation/assign_provider_action/jsonschema/1-0-0

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for tracking what options are availble when assigning a provider",
+	"self": {
+		"vendor": "com.fieldnation",
+		"name": "assign_provider_action",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	
+	"type": "object",
+	"properties": {
+        "provider_id": {
+            "type": "number"
+        },
+        "available_providers": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "presented_providers": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "requested_providers": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "declined_providers": {
+            "type": "array",
+            "items": {
+                "type": "number"
+            }
+        },
+        "view": {
+            "enum": ["list", "map"]
+        }
+	},
+	"required": ["provider_id", "available_providers", "presented_providers", "requested_providers", "declined_providers", "view"],
+	"additionalProperties": false
```

Comment:
Bit confused here. Don't we need to pass additional context here like workOrder, userInterface etc.?